src/torrent.cpp

```diff
@@ -5878,6 +5887,13 @@ namespace {
 		}
 	}
 
+    void torrent::set_ssl_ca_cert_buffer(std::string const& certificate)
```

Comment:
again, I don't see how it can be safe to *change* this on a torrent.

Reply:
Sorry my code was not finalised yet...